PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/build.gradle

```diff
@@ -0,0 +1,14 @@
+intellij {
+    plugins = ['java', 'maven', 'maven-model', 'gradle']
+}
+
+dependencies {
+    compileOnly project(path: ':azure-intellij-plugin-lib')
+    runtimeOnly project(path: ':azure-intellij-plugin-lib', configuration: 'instrumentedJar')
+    compileOnly project(path: ':azure-intellij-plugin-lib-java')
+    runtimeOnly project(path: ':azure-intellij-plugin-lib-java', configuration: 'instrumentedJar')
+    implementation 'com.microsoft.azure:azure-toolkit-common-lib'
+    implementation 'com.microsoft.azure:azure-toolkit-ide-common-lib'
+    aspect 'com.microsoft.azure:azure-toolkit-common-lib'
```

Comment:
this is only needed if you use our own telemetry framework.

Reply:
I will keep this for now as I want to experiment with using your telemetry framework but with a different endpoint.